File: lambda_function.js
var lamvery = require('./lamvery.js');
var fs = require('fs');

exports.lambdaHandler = function(event, context) {
    lamvery.env.load();
    console.log(event);
    console.log(process.env);
    lamvery.secret.get('foo', function(err, data) {
        console.log(data);
        lamvery.secret.file('bar.txt', function(err, p) {
            fs.readFile(p, 'utf-8', function(err, txt) {
                console.log(txt);
                context.succeed("done.");
            });
        });
    });
}

File: lambda_function.py
import lamvery
import os


def lambda_handler(event, context):
    lamvery.env.load()
    print(os.environ)
    print(context)
    print(lamvery.secret.get('foo'))
    print(open(lamvery.secret.file('bar.txt'), 'r').read())
    return event
